packages/dapp/src/components/Widgets/Widgets.style.tsx

```diff
@@ -84,6 +84,10 @@ export const WidgetContainer = styled(Box, {
     [`@media screen and (min-height: 700px)`]: {
       marginTop: showWelcome ? 'calc( 50vh - 680px / 2.75 - 80px)' : '0', // (mid viewheight - half-two/thirds widget height - header height )
     },
+
+    [`@media screen and (min-height: 900px)`]: {
```

Comment:
Should we use standard breakpoint numbers?
https://mui.com/material-ui/customization/breakpoints/

Reply:
@chybisov I agree! Good, you´re asking - but how does that apply if looking at min-heights? In this approach, I am positioning the widget for the welcome-screen because of limited vertical space. I´d also prefer using as little of those "custom" breakpoints as possible.

Reply:
Oh, didn't notice that's about the height 😄